src/components/Pill.svelte

```diff
@@ -0,0 +1,12 @@
+<script>
+  export let data;
```

Comment:
"message" is more concrete

```suggestion
  export let message = "";
```